src/main/java/com/neighborhoodAssurance/NeighborController.java

```diff
@@ -21,21 +16,8 @@ public String index()
 	@ResponseBody
 	public String readJSON() 
 	{
-		/*
-		 * DTO Information
-		 */
-		return "index"; //return DTO
-	}
-	/*@RequestMapping(value="/index", method=RequestMethod.GET)
-	public String readSearchValue(@RequestParam(value="searchTerm", required=true)String searchTerm) //May need defaultValue="
-	{  
-		/*
-		 * DTO Information?
-		 

```

Comment:
Good idea... remove commented out code.  Avoid committing and pushing commented-out code to version control because  1)  You can always go back in history to see what used to be there.  2) If you're trying an experimental feature, you can do it in a branch, and not impact master.